### PR TITLE
Add FLIX CLI docs

### PR DESCRIPTION
**Pull request by chasefleming**

### CODE REVIEW

File: docs/tooling/flow-cli/flix.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+```shell
+flow flix <query> [<argument> <argument>...] [flags]
+```
+
+Queries can be a FLIX id, name, or path to a local FLIX file.
+
+## Example Usage
+
+```shell
+# Execute a FLIX script by id on Testnet
```
Comment (sideninja) [inline docs/tooling/flow-cli/flix.md:19]: very NIT: but I feel the example with name should go first as it feels more natural to understand the command. Also the ID should be in correct format hex encoded ID

File: docs/tooling/flow-cli/flix.md [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+```
+
+```shell
+# Execute a FLIX transaction by name on Testnet
+flix flow transfer-flow 5.0 "0x123" --network testnet --signer "testnet-account"
+```
+
+```shell
+# Execute a local FLIX script by path on Testnet
+flow flix path ./multiply.template.json 2 3 --network testnet`
```
Comment (sideninja) [inline docs/tooling/flow-cli/flix.md:30]:
```suggestion
flow flix ./multiply.template.json 2 3 --network testnet`
```

File: docs/tooling/flow-cli/flix.md [hunk 3]
```diff
@@ -0,0 +1,194 @@
+---
+title: Execute Flow Interaction Templates (FLIX) with the Flow CLI
+sidebar_title: Execute FLIX
+description: How to execute Flow Interaction Templates (FLIX) on Flow from the command line
+sidebar_position: 12
+---
+
+The Flow CLI provides a command to execute Flow Interaction Templates (FLIX).
```
Comment (sideninja) [inline docs/tooling/flow-cli/flix.md:8]: Can we add few words what is FLIP (you can even copy-paste from FLIX doc)

**Comment by chasefleming** [comment]
@sideninja Feedback addressed. Ready for another review.

**Pull request merged**